test/unit/presenters/areas_presenter_test.rb

```diff
@@ -6,50 +6,123 @@ class AreasPresenterTest < ActiveSupport::TestCase
     setup do
       bridge = OpenStruct.new(:payload => {
         :code => 200,
-        :areas => [ { "id" => 123, "name" => "Westminster City Council", "country_name" => "England", "type" => "LBO" },
-                    { "id" => 234, "name" => "London", "country_name" => "England", "type" => "EUR" } ]
+        :areas => [
+          {
+            "id" => 123,
+            "name" => "Westminster City Council",
+            "country_name" => "England",
+            "type" => "LBO",
+            "codes" => {
+              "gss" => "E09000033",
+            },
+          },
+          {
+            "id" => 234,
+            "name" => "London",
+            "country_name" => "England",
+            "type" => "EUR",
+            "codes" => {
+              "gss" => "E15000007",
+            },
+          },
+        ],
       })
       @presenter = AreasPresenter.new(bridge)
     end
+
     should "format the correct response status" do
       assert_equal "ok", @presenter.present["_response_info"]["status"]
     end
-    should "expose the correct data" do
-      assert_equal "westminster-city-council", @presenter.present["results"].first["slug"]
-      assert_equal "Westminster City Council", @presenter.present["results"].first["name"]
-      assert_equal "England", @presenter.present["results"].first["country_name"]
-      assert_equal "LBO", @presenter.present["results"].first["type"]
-      assert_equal "london", @presenter.present["results"].last["slug"]
-      assert_equal "London", @presenter.present["results"].last["name"]
-      assert_equal "England", @presenter.present["results"].last["country_name"]
-      assert_equal "EUR", @presenter.present["results"].last["type"]
 
-      refute @presenter.present["results"].first.has_key?("parent_area")
+    context "first result" do
+      setup do
+        @result = @presenter.present["results"].first
+      end
+
+      should "expose the correct data" do
+        assert_equal "westminster-city-council", @result["slug"]
+        assert_equal "Westminster City Council", @result["name"]
+        assert_equal "England", @result["country_name"]
+        assert_equal "LBO", @result["type"]
+        assert_equal "E09000033", @result["gss"]
+
+        refute @result.has_key?("parent_area")
+      end
+    end
+
+    context "second result" do
+      setup do
+        @result = @presenter.present["results"][1]
+      end
+
+      should "expose the correct data" do
+        assert_equal "london", @result["slug"]
+        assert_equal "London", @result["name"]
+        assert_equal "England", @result["country_name"]
+        assert_equal "EUR", @result["type"]
+        assert_equal "E15000007", @result["gss"]
+      end
     end
   end
+
   context "presenting a postcode lookup mapit response" do
```

Comment:
@jennyd, I've added a commit removing a `context` and renaming the remaining one